docs/docs/event-bus.md

```diff
@@ -5,7 +5,8 @@ title: Event Bus
 # Event Bus
 
 Marathon has an internal event bus that captures all API requests and scaling events. This is useful for integrating with load balancers, keeping stats, etc.
```

Comment:
Could we briefly explain how this is useful? The stats use makes immediate sense, but the load balancer use case could use some explanation.